finagle-redis/src/test/scala/com/twitter/finagle/redis/commands/script/ScriptClientIntegrationSuite.scala

```diff
@@ -0,0 +1,364 @@
+package com.twitter.finagle.redis.integration
+
+import java.math.BigInteger
+
+import com.twitter.finagle.redis.naggati.RedisClientTest
+import com.twitter.finagle.redis.{Client, ServerError}
+import com.twitter.io.Charsets
+import com.twitter.util.Await.result
+import com.twitter.util.Future
+import org.jboss.netty.buffer.{ChannelBuffer, ChannelBuffers}
+import org.junit.runner.RunWith
+import org.scalatest.junit.JUnitRunner
+import org.scalatest.MustMatchers
+
+// Testing scripting features of redis Client
+@RunWith(classOf[JUnitRunner])
+class ScriptClientIntegrationSuite extends RedisClientTest with MustMatchers {
```

Comment:
please don't use MustMatchers unless you get something really cool out of it, in scalatest 2.x the assert macro means we can get really good error messages easily out of assert.